site/content/en/docs/Guides/Client SDKs/unreal.md

```diff
@@ -39,6 +39,6 @@ Available settings:
 
 - Health Ping Enabled. Whether the server sends a health ping to the Agones sidecar. (default: `true`)
 - Health Ping Seconds. Interval of the server sending a health ping to the Agones sidecar. (default: `5`)
-- Debug Logging Enabled. Debug logging for development of this Plugin. (default: `false`)
 - Request Retry Limit. Maximum number of times a failed request to the Agones sidecar is retried. Health requests are not retried. (default: `30`)
+- Send Ready at Startup. Automatically send a Ready request when the server starts. Disable this to manually control when the game server should be marked as ready. (default: `true`)
```

Comment:
Does there need to be more docs around how to use the the supplied methods, such as `Allocate`, `Ready`, `SetLabel` etc?

Reply:
I tend to agree, it might be worth adding a paragraph on how these mehtods / structs are available to be set from within UE4Editor?

Reply:
Might also be worth adding a paragraph around using this locally ... eg. how we can turn off the healthcheck/ready checks when just launching via UE4Editor that makes it easier for most GameDevs to iterate fast before pushing into our clusters.

Reply:
Have added some docs on how to use the sdk methods

Reply:
For local dev (or for running outside an agones environment) I think the plugin could use some more work to support disabling executing the sdk methods / auto health checks. Was thinking about an additional config value `bAgonesEnabled` which can be set to `default`, `forceEnabled`, or `forceDisabled`, where `default` will disable calling out to the agones sidecar when running locally inside the unreal editor